pkg/controller/servicebindingrequest/nested/nested.go

```diff
@@ -0,0 +1,188 @@
+package nested
```

Comment:
the function in this file look very like as good candidate for an util map package, how they are specific to anything nested?

Reply:
Because it is designed to extract and produce nested data structures.